config/config.php

```diff
@@ -22,7 +22,6 @@
     Laminas\I18n\ConfigProvider::class,
     Laminas\Validator\ConfigProvider::class,
     Phly\EventDispatcher\ConfigProvider::class,
-    Phly\Swoole\TaskWorker\ConfigProvider::class,
```

Comment:
Removed this package, as current mezzio-swoole versions have the functionality incorporated in them.